### ISSUE TITLE
Instrumentation seems to be broken on latest BullMQ version

### ISSUE THREAD

**Issue by unflxw** [appsignal/opentelemetry-instrumentation-bullmq#15]
https://github.com/appsignal/opentelemetry-instrumentation-bullmq/actions/runs/12312736719

**Comment by raine** [comment]
Would this explain why I'm not seeing any spans being generated by this library?

**Comment by unflxw** [comment]
Yes, if you're using `bullmq@5`, that would be why. You may want to try using BullMQ's built-in OpenTelemetry support: https://docs.bullmq.io/guide/telemetry/getting-started

**Comment by raine** [comment]
Yep, I figured out later that you can pass `telemetry` also to `FlowProducer` options. (Not that well documented.)

Initially it looked like adding traces to FlowProducers is only possible through this library.

**Comment by unflxw** [comment]
Fixed in #17. Closing.
